run_tests.sh

```diff
@@ -0,0 +1,60 @@
+# Copyright 2024 10xEngineers
+
+#!/bin/bash
+
```

Comment:
This script has a lot of missing safety checks, make sure this script compiles in https://www.shellcheck.net/. Paste a photo here of it passing.

Also add set `-euo pipefail`
and add the `exit 0` command at the end, and do error checking, so that the exit code is not always 0, if there is an error, it is checked and reported, at the end of the script

Reply:
updated script as per the comments.
Here's the screenshot of shellchecker: 
![Screenshot from 2024-07-30 13-01-11](https://github.com/user-attachments/assets/267a7b05-bf32-47fb-9bc0-b074a0cf4e75)

We can successfully ignore the "SC2329" error. [(reference)](https://www.shellcheck.net/wiki/SC2329)
![Screenshot from 2024-07-30 13-00-06](https://github.com/user-attachments/assets/7e8bcd47-378e-439c-9f54-ae44a1bee8f3)